src/components/landing/header/Header.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styles from './Header.module.scss';
+import { Button } from 'primereact/button';
+import Link from 'next/link';
+import useCurrentUser from '@hooks/useCurrentUser';
+import Image from 'next/image';
+import Flip from 'react-reveal/Flip';
+import logo from '@utils/images/fin-logo.svg';
+
+const Header = () => {
+  const currentUser = useCurrentUser();
+  return (
+    <section id={styles.header}>
+      <ul className={styles.circles}>
+        {[...Array(10)].map((i) => (
```

Comment:
This counter is not working as expected and throw an error in the console. Only need a small modification to work:  `[...Array(10).keys()].map((i) => (`

Reply:
You are right, it is ready.